### PR TITLE
Aastha_TreeProgram_15mins

### PR DESCRIPTION
**Pull request by AasthaJJun20**
Capture

### CODE REVIEW

File: src/aasthaJJun20/TreeProgram.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package aasthaJJun20;
+//[3,5,5,11,9,10,13] --> 4
+
+public class  TreeProgram{
+	void displayVisibleTrees(int[] input) {
+		int count = 0, tallest = 0;
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/TreeProgram.java]: tallest = input[0]

Comment (KrishnaKTechnocredits) [reply src/aasthaJJun20/TreeProgram.java]: count = 1, tallest = input[0]

File: src/aasthaJJun20/TreeProgram.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+		for(int index = 0; index < input.length; index++) {
```
Comment (KrishnaKTechnocredits) [inline src/aasthaJJun20/TreeProgram.java]: index = 1

**Review by harshvegada — APPROVED** [review]
Code changes as per the comments

**Pull request merged**